src/sentry/static/sentry/app/components/events/interfaces/spans/index.tsx

```diff
@@ -66,4 +174,43 @@ const StyledSearchBar = styled(SearchBar)`
   margin-bottom: ${space(1)};
 `;
 
-export default SpansInterface;
+const AlertMessageContainer = styled('div')`
+  margin-bottom: ${space(1)};
+`;
+
+function filterSpansWithErrors(
+  parsedTrace: ParsedTraceType,
+  tableData: TableData | null | undefined
+): TableData | null | undefined {
+  if (!tableData) {
+    return tableData;
+  }
+
+  const data = tableData?.data ?? [];
+
+  const filtered = data.filter(row => {
+    const spanID = row['trace.span'] || '';
+
+    if (!spanID) {
```

Comment:
nitpick: You sometimes use `thingID` and `thingId` could we normalize on one form?

Reply:
https://github.com/getsentry/sentry/pull/18025/commits/3b297a59d9fc7cea380b3c80b26cfa2eed36e9cb